.github/workflows/code-quality-review.yml

```
@@ -10,15 +10,15 @@ jobs:
   claude-review:
     # Run on PR events, or when someone comments "@claude review" on a PR
     if: |
-      github.event_name == 'pull_request' ||
+      github.event_name == 'pull_request_target' ||
       (github.event_name == 'issue_comment' && 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)